### DISCUSSION TITLE
[Bug]:

### DISCUSSION THREAD

**Discussion by GaneshKandu** [pimcore/pimcore#11489] [Q&A / Support] [answered]
### Expected behavior

I have installed Pimcore10.3.1 but i am not able to edit pages
highlight editable is alsp not working

![screenshot-webuzo web-2022 02 18-16_04_22](https://user-images.githubusercontent.com/12833130/154666450-bbc8df3c-cd92-46c1-aa56-e1d38b9dfe13.png)

### Actual behavior

i should able to edit default page and see highlight editable

### Steps to reproduce

I followed step 1 to 5

[Installation - Pimcore](https://pimcore.com/docs/pimcore/current/Development_Documentation/Getting_Started/Installation.html)

**Comment by dvesh3** [comment, marked as answer]
This is the default template with no editables on it. You have to provide a template which defines editables that you can edit in the editmode. https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/index.html Thanks!

**Reply by GaneshKandu** [reply]
Thanks !
